content/sensu-core/1.6/reference/checks.md

```diff
@@ -301,6 +298,64 @@ value), _and_ the Sensu client definition does not have a matching definition
 attribute, a [check result][4] indicating "unmatched tokens" will be published
 for the check execution (e.g.: `"Unmatched check token(s): disk.warning"`).
 
+#### Token data type limitations
+
+As part of the substitution process, Sensu converts all tokens to strings.
+This means that tokens cannot be used for standalone integer values or to access individual list items.
+
+For example, token substitution **cannot** be used for a check interval because the interval attribute requires an _integer_ value. But token substitution **can** be used for alerting thresholds since those values are included within the command _string_.
```

Comment:
```suggestion
For example, token substitution **cannot** be used for specifying a check interval because the interval attribute requires an _integer_ value. But token substitution **can** be used for alerting thresholds since those values are included within the command _string_.
```